README.md

```diff
@@ -1,42 +1,34 @@
 Using the repository template
 =============================
 
-
-
 #### Pre-requisites:
  - Mac OS or Linux OS
+ - [Python 2.X](https://www.python.org)
  - [Stata MP](http://www.stata.com/statamp/)
-    - Executable should also be added to the `PATH` environment variable so it is callable from the command line with `statamp`
  - [R](https://www.r-project.org/)
-    - Executable should also be added to the `PATH` environment variable so it is callable from the command line with `Rscript`
- - [scons](http://scons.org/) 
  - [Lyx](https://www.lyx.org/)
-    - Executable should also be added to the `PATH` environment variable so it is callable from the command line with `lyx`
+    - For the above four, executables need to added to `path` environment variable. So they are callable from the command line with `python`, `statamp`, `Rscript`, and `lyx`.
+ - [SCons](http://scons.org/) version 2.4.0 or later
```

Comment:
Change to (since one can use version 2.4.0 or below and still run the repo):

- [SCons](http://scons.org/)
  - version 2.4.0 or later is best if using the cache.